scripts/test_ports/cmake/portfile.cmake

```
@@ -16,7 +16,7 @@ vcpkg_configure_cmake(
     OPTIONS
         -DBUILD_TESTING=OFF
         #-DCMAKE_USE_SYSTEM_LIBRARIES=ON
-        -DCMAKE_USE_SYSTEM_LIBARCHIVE=ON
+        -DCMAKE_USE_SYSTEM_LIBARCHIVE=OFF # CMake is not compatible with libarchive 3.5.1
```

```suggestion
        -DCMAKE_USE_SYSTEM_LIBARCHIVE=ON
```
the build logs in 8a00f62c720795ea0587d68af072448afa6f7e61 doesn't support that.